ios/Classes/Bindings/Models/ParticipantDTO.swift

```diff
@@ -37,7 +37,7 @@ extension DTO {
             case "UNKNOWN": participantType = .none
             case "USER": participantType = .user
             case "LISTENER": participantType = .listener
-            default: fatalError("TODO: Throw actual error here")
+            default: fatalError(.decoderFailed)
```

Comment:
We can **throw** an error here. And the same can be done in other functions which are similar.